presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3000,6 +3000,19 @@ public void testOrderByWithSimilarExpressions()
                 "SELECT 1, 172799.49, 1");
     }
 
+    @Test
+    public void testComplexOrderBy()
```

Comment:
nit: the `order by` here doesn't seem to be complex; actually it is quite simple; what makes it special is that appears below multiple layers of projections; I'm wondering if test name could capture this specific better; testOrderByUnderManyProjections?